packages/frontend/src/Happy/Frontend/Mangler.lhs

```diff
@@ -413,6 +284,10 @@ So is this.
 
 > checkRules [] _ nonterms = return (reverse nonterms)
 
+> getErrorHandlerType' :: [Directive t] -> ErrorHandlerType
+> getErrorHandlerType' = conv . getErrorHandlerType where
+>   conv ErrorHandlerTypeExpList' = ErrorHandlerTypeExpList
+>   conv ErrorHandlerTypeDefault' = ErrorHandlerTypeDefault
```

Comment:
Ah, so there seems to be another type `ErrorHandlerType`. Why? Also `getErrorHandlerType'` returns the type without prime, while `getErrorHandlerType` returns the type with prime... Unexpected

Reply:
You're right, the `ErrorHandlerType'` type was useless, I introduced it to break a dependency cycle, but that's actually not required.